tests/tensorflow/r2.10/classifier-efficientnet.libsonnet

```diff
@@ -163,8 +163,8 @@ local tpus = import 'templates/tpus.libsonnet';
     efficientnet + v2_8 + functional + common.tpuVm,
     efficientnet + v3_8 + functional,
     efficientnet + v3_8 + hbm + mixins.Unsuspended,
-    efficientnet + v2_8 + convergence + timeouts.Hours(45) + { schedule: '0 8 * * 2,5' },
-    efficientnet + v3_8 + convergence + timeouts.Hours(45) + { schedule: '0 8 * * 2,5' },
+    efficientnet + v2_8 + convergence + timeouts.Hours(45) + { schedule: null },
```

Comment:
The default schedule is already `null`, so you can just remove these overrides completely